src/dune/lib.ml

```diff
@@ -397,61 +397,63 @@ let hash t = Id.hash (to_id t)
 
 include Comparable.Make (T)
 
-let link_flags t (mode : Link_mode.t) (lib_config : Lib_config.t) =
-  let archive_deps_and_flags =
+module Link_params = struct
+  type t =
+    { include_dirs : Path.t list
```

Comment:
I'd leave a comment here. The `deps` field isn't particularly clear.

Reply:
Indeed, I added a comment